apps/courses-site/tsconfig.json

```diff
@@ -25,7 +25,9 @@
     "**/*.jsx",
     "../../apps/courses-site/.next/types/**/*.ts",
     "../../dist/apps/courses-site/.next/types/**/*.ts",
-    "next-env.d.ts"
+    "next-env.d.ts",
+    ".next/types/**/*.ts",
```

Comment:
This is something to look into but not a blocker:

Right now this would add vite config into the main tsconfig not just testing

Reply:
I think nx added this for me so will move thanks :)